<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget2048</class>
 <widget class="QWidget" name="Widget2048">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>520</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget2048</string>
  </property>
  <widget class="QLabel" name="label_score">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_best">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>BEST</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_best">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_score">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);</string>
   </property>
   <property name="text">
    <string>SCORE</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new_game">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>New Game</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
